cmd/kubeadm/app/cmd/util/cmdutil.go

```diff
@@ -141,3 +141,12 @@ func GetClientSet(file string, dryRun bool) (clientset.Interface, error) {
 	}
 	return kubeconfigutil.ClientSetFromFile(file)
 }
+
+// GetFromConfigOrFlags checks if the "name" flag has been set, returns the value of flag on yes, otherwise returns the value from config.
+func GetFromConfigOrFlags(flagSet *pflag.FlagSet, name string, cfgValue interface{}, flagValue interface{}) interface{} {
```

Comment:
```suggestion
func ValueFromFlagsOrConfig(flagSet *pflag.FlagSet, name string, cfgValue interface{}, flagValue interface{}) interface{} {
```

proposing this new name, to explain we obtain a value and that flags have priority.